Build Terra Station steps not working

# Summary
After doing ["Build Terra Station" instructions](https://github.com/terra-project/station#build-terra-station) from the Readme 

> git clone https://github.com/terra-project/station.git
cd station
cp src/chains.sample.ts src/chains.ts
yarn
yarn start

I've got error: 
`Failed to compile. Module not found: Can't resolve '../../use-station/src'`

![image](https://user-images.githubusercontent.com/987517/111670789-4655d900-8829-11eb-8de8-62cb31436e51.png)


# Expected
Terra Station should now be running locally at https://local.terra.money:3000.

# Environments
* Stage: development
* Chain: *
* OS: MacOS
* Platform: Terra Station

# Reproduce
1. Run commands from ["Build Terra Station" instruction](https://github.com/terra-project/station#build-terra-station) from the Readme 

P.s. I understood that **MAYBE** I need to put another project "use-station" to one of subfolders but I think such information must be presented in the Build instruction.

--- a/README.md
+++ b/README.md
@@ -11,7 +11,6 @@ Terra Station allows users to:
 - A dashboard monitoring key Terra macroeconomic variables
 - Atomically swap currencies on the Terra network at the effective on-chain exchange rate.
 
-
 ## Running Terra Station with useStation locally
 
 This guide explains how you can set up Terra Station repositories for local development.
@@ -25,18 +24,25 @@ Add the following entry to your `/etc/hosts` file:
 ```
 127.0.0.1 local.terra.money
 ```
-​
+
 ### Build Terra Station
 
 ```sh
 git clone https://github.com/terra-project/station.git
 cd station
 cp src/chains.sample.ts src/chains.ts
+git submodule init
+git submodule update
 yarn
 yarn start
 ```
-​
+
+> :warning: For Windows user, you need to change the `SASS_PATH` inside your `.env` file.
+> The value must be `SASS_PATH=./node_modules;./src/styles`
+
 Terra Station should now be running locally at https://local.terra.money:3000.
+
+> :mortar_board: You need to change your host file to be able to access `local.terra.money` locally.
 
 #### Customizing [useStation](https://github.com/terra-project/use-station)
 
@@ -74,7 +80,6 @@ yarn start
 ​
 An Electron app should now be running against https://local.terra.money:3000.
 You can now launch the app version of Terra Station.
-
 
 ## Available Scripts
 

--- a/package.json
+++ b/package.json
@@ -82,6 +82,7 @@
     "@types/secp256k1": "^4",
     "@types/semver": "^7.3.4",
     "@types/socketcluster-client": "^13",
+    "cross-env": "^7.0.3",
     "husky": "^4.3.7",
     "lint-staged": "^10.5.3",
     "prettier": "^2",
@@ -100,7 +101,7 @@
     "build-storybook": "build-storybook -s public",
     "analyze": "source-map-explorer 'build/static/js/*.js'",
     "build-extension": "webpack --config extension/webpack.config.js",
-    "ext": "REACT_APP_ENV=extension INLINE_RUNTIME_CHUNK=false react-app-rewired build && npm run build-extension"
+    "ext": "cross-env REACT_APP_ENV=extension INLINE_RUNTIME_CHUNK=false react-app-rewired build && npm run build-extension"
   },
   "eslintConfig": {
     "extends": "react-app",

